language/transaction-builder/generator/src/common.rs

```diff
@@ -51,7 +51,8 @@ fn quote_parameter_as_field(arg: &ArgumentABI) -> Named<Format> {
     }
 }
 
-pub(crate) fn make_abi_enum_container(abis: &[ScriptABI]) -> ContainerFormat {
+// TODO: Update to hanlde script function ABIs
```

Comment:
typo: hanlde

Reply:
Also, it seems that all these TODOs would be more useful (e.g. searchable) if they included a github issue (like this `TODO(#1234)`).

Reply:
Possibly. I'm not sure if I want to add these issues though as the next PR that I'm working on on top of this fixes all of these? What are your thoughts?